tests/Http/AuthenticatorTest.php

```diff
@@ -43,4 +50,189 @@ public function testSetsAndProvidesCredentials()
 
         $this->assertSame($credentials, $authenticator->getAuthCredentials());
     }
+
+    public function testTokensAreRefreshedWithoutCallbacks()
+    {
+        $authType = \Mockery::mock(Auth::class, [
```

Comment:
Since Mockery is imported on line 16, we can probably do a quick find & replace for `\Mockery..`, `Mockery..` in this test file.

Reply:
Done!